wvd-templates/Update existing WVD host pool/mainTemplate.json

```diff
@@ -148,93 +141,100 @@
                 "SSD",
                 "HDD"
             ],
-            "type": "String",
+            "type": "string",
             "metadata": {
                 "description": "The VM disk type for the VM: HDD or SSD."
             }
         },
         "rdshVmSize": {
             "defaultValue": "Standard_D2s_v3",
-            "type": "String",
+            "type": "string",
             "metadata": {
                 "description": "The size of the session host VMs."
             }
         },
         "rdshUseManagedDisks": {
             "defaultValue": true,
-            "type": "Bool",
+            "type": "bool",
             "metadata": {
                 "description": "True indicating you would like to use managed disks or false indicating you would like to use unmanaged disks."
             }
         },
         "storageAccountResourceGroupName": {
             "defaultValue": "",
-            "type": "String",
+            "type": "string",
             "metadata": {
                 "description": "(Required when rdshUseManagedDisks = False) The resource group containing the storage account of the image vhd file."
             }
         },
+        "storageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "The Name of the Storage Account(Unique Name) to store unamanged disks for unmanaged gallery VM"
```

Comment:
Please use the same description that I provided in the "Create and provision..." review.